siac/rentercmd.go

```diff
@@ -110,21 +118,81 @@ func abs(path string) string {
 	return abspath
 }
 
-func renterdownloadqueuecmd() {
+func renteruploadscmd() {
```

Comment:
None of the command functions (not just renter) have docstrings. I don't think they're necessary as the functions are very well described in the command initialization. If anything I'd say the only docstring needed would be something like "renteruploadscmd is the handler for the command renter uploads" but that's already obvious from the name of the function.